test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -272,7 +274,7 @@ class ApiFuzzer : public BasicFuzzer {
   grpc_completion_queue* cq_ = nullptr;
   grpc_server* server_ = nullptr;
   grpc_channel* channel_ = nullptr;
-  grpc_resource_quota* resource_quota_;
+  grpc_core::RefCountedPtr<grpc_core::ResourceQuota> resource_quota_ = nullptr;
```

Comment:
No need for `= nullptr`; that's the default.

Reply:
Good catch, fixed.